scripts/ci/upload-release

```diff
@@ -65,7 +65,7 @@ RELEASE_NAME="${RELEASE_NAME}" \
 
 # Upload source maps to sentry.io
 SENTRY_AUTH_TOKEN="${SENTRY_AUTH_TOKEN}" \
-  RELEASE_NAME="${RELEASE_NAME}" \
+  RELEASE_NAME="${PKG_TARGET}+${PKG_VERSION}" \
```

Comment:
why was it important?

Reply:
check sentry UI, we uploaded sourcemaps for `master+dcos-ui+v1.23.4` but reported to `master+v1.23.4`… 🤷‍♂️

Reply:
`RELEASE_NAME` is only used as filename, not as release version string (we might want to get rid of that different name at all 🤔 )